lib/membrane/aac/parser.ex

```diff
@@ -5,7 +5,7 @@ defmodule Membrane.AAC.Parser do
   Supports both plain and ADTS-encapsulated output (configured by `out_encapsulation`).
   Input with encapsulation `:none` is supported, but correct AAC caps need to be supplied with the stream.
 
-  Adds sample rate based timestamp to metadata if absent.
+  Puts sample rate based PTS into buffer if absent.
```

Comment:
I'd rephrase it, WDYT of:
```suggestion
  If PTS is absent, it calculates and puts one based on the sample rate.
```